frontend/src/main/components/Nav/Footer.js

```diff
@@ -1,8 +1,16 @@
+import { useEffect, useState } from "react";
 import { Container } from "react-bootstrap";
-
 export const space = " ";
 
 export default function Footer() {
+  const [repoUrl, setRepoUrl] = useState("");
+  useEffect(() => { 
+    fetch("/api/systemInfo")
+      // .then((res) => res.json())
+      .then((data) => {
+        setRepoUrl(data.sourceRepo);
+      });
+  }, []);
```

Comment:
Suggest removing all of this, and instead, pass systemInfo into the component in BasicLayout.js.